contrib/olivere/elastic/elastictrace_test.go

```diff
@@ -149,6 +153,7 @@ func checkPUTTrace(assert *assert.Assertions, mt mocktracer.Tracer) {
 	assert.Equal("PUT /twitter/tweet/?", span.Tag(ext.ResourceName))
 	assert.Equal("/twitter/tweet/1", span.Tag("elasticsearch.url"))
 	assert.Equal("PUT", span.Tag("elasticsearch.method"))
+	assert.Equal(`{"user": "test", "message": "hello"}`, span.Tag("elasticsearch.body"))
```

Comment:
Is there a case that checks the peek works after an error?

Reply:
I've just assumed that it wasn't needed, since we already test `peek`. But I've added two more tests now, one tests that error is cut off correctly and the other that the request body is cut off correctly.

Reply:
Well we still use peek when we got a response, I didn't expect a test with the actual cut, but I wanted to make sure that the response still were traced. Either way, more test cases is good.